.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Fancy json that accepts comments? what is this witchcraft?

Reply:
default vscode thing idk